src/math/monte_carlo_integration.jl

```diff
@@ -0,0 +1,16 @@
+"""
+Monte carlo integration is a very easy and scalable way to do multidimentional integrals.
+However, only single variable integrals is considered.
+
+Refereces:
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+
+Contributed by: [Ved Mahajan](https://github.com/Ved-Mahajan)
+"""
+function monte_carlo_integration(f,N,span)
```

Comment:
Please add documentation for this function.

Reply:
done :+1:

Reply:
Hmm, I think it'd be better if you do it as the [`knapsack`](https://github.com/TheAlgorithms/Julia/blob/main/src/knapsack/knapsack.jl) problem. I'm not really sure if the Julia documenter is going to work like this. @arubhardwaj?

Reply:
What do you think is missing?

IMO, the docs has all that is needed to use that function.

Reply:
Once again, we should probably merge this and think about the general presentation we want for this repo.